runtime/slot.js

```diff
@@ -48,8 +48,14 @@ class Slot {
   }
   startRender() {
     if (this.startRenderCallback) {
-      let contentTypes = this.constructRenderRequest();
-      this.startRenderCallback({particle: this.consumeConn.particle, slotName: this.consumeConn.name, contentTypes});
+      const slotName = this.consumeConn.name;
+      const particle = this.consumeConn.particle;
+      const context = this.getContext();
+      if (context.updateParticleName) {
```

Comment:
Maybe this is too much of a hack? This is to deal gracefully with whatever cases can result in our having a context here that's not a straight `DomContext`. Currently I know there's a `SetDomContext` and also some tests seem to pass `manifest`, and empty objects. If we'd prefer to do this differently am of course open to revision.

This PR is envisioned as a first step to restore illumination and move the slot/particle instrumentation needed into runtime, and I figure we can add sets, inner slots, etc. subsequently.